core-web/libs/block-editor/src/lib/extensions/ai-content-actions/ai-content-actions.component.scss

```
@@ -0,0 +1,43 @@
+@use "variables" as *;
+
+.ai-actions-list {
+    width: 12.5rem;
+    padding: $spacing-1;
+    margin-left: $spacing-8;
+    background-color: $white;
+
+    // list option
+    .p-listbox-item {
+        background-color: $white;
+        padding: $spacing-2 $spacing-3;
+
+        // accept option
+        &:first-child {
+            background-color: var(--color-palette-primary-200);
+        }
+
+        // regenerate option
+        &:nth-child(2) {
+            padding: $spacing-3;
+        }
+
+        // delete option
+        &:last-child {
+            border-top: 1px solid $color-palette-gray-300 !important;
+        }
+
+        &:hover {
+            background-color: var(--color-palette-primary-100);
```

Try to use `sass` variables here

```suggestion
            background-color: $color-palette-primary-100;
```